default/scripting/ship_parts/FighterHangar/FT_HANGAR_4.focs.txt

```diff
@@ -4,13 +4,16 @@ Part
     exclusions = [ "FT_HANGAR_0" "FT_HANGAR_1" "FT_HANGAR_2" "FT_HANGAR_3" ]
     class = FighterHangar
     capacity = 1
-    damage = 10
-    combatTargets = OrderedAlternativesOf [
-        And [ [[COMBAT_TARGETS_VISIBLE_ENEMY]]  [[COMBAT_TARGETS_NOT_DESTROYED_SHIP]] ]
-        And [ [[COMBAT_TARGETS_VISIBLE_ENEMY]]  Fighter ]
+    damage = 12
+    combatTargets = And [
+        [[COMBAT_TARGETS_VISIBLE_ENEMY]]
+        Or [
+           [[COMBAT_TARGETS_NOT_DESTROYED_SHIP]]
+           [[COMBAT_TARGETS_PLANET_WITH_DEFENSE]]
+        ]
     ]
     mountableSlotTypes = Internal
-    buildcost = 30 * [[FLEET_UPKEEP_MULTIPLICATOR]] * [[SHIP_PART_COST_MULTIPLIER]]
+    buildcost = 50 * [[FLEET_UPKEEP_MULTIPLICATOR]] * [[SHIP_PART_COST_MULTIPLIER]]
```

Comment:
50 is too high, 2x the other fighters. Make it 25?

Reply:
agreed

Reply:
This should be fixed. Are we done for the moment?